.drone/drone.jsonnet

```diff
@@ -323,9 +315,9 @@ local manifest(apps) = pipeline('manifest') {
       path: 'loki',
     },
     steps: [
-      make('test', container=false) { depends_on: ['clone'] },
-      make('lint', container=false) { depends_on: ['clone'] },
       make('check-generated-files', container=false) { depends_on: ['clone'] },
+      make('test', container=false) { depends_on: ['clone','check-generated-files'] },
+      make('lint', container=false) { depends_on: ['clone','check-generated-files'] },
```

Comment:
make test and lint depend on check-generated-files so they wait for it to finish as it will rebuild protos